### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bamse16**
Changed config path for macOS

### CODE REVIEW

**Review by iosdevben — COMMENTED** [review]
The following is not a valid path. 

"$(brew --prefix)"/user/local/

**Comment by bamse16** [comment]
Should be `/usr/local/` not `/user/local/`.

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ brew install buildkite/buildkite/buildkite-agent
 You then need to configure your agent token, which you can find your agent token on your "Agents" page in Buildkite:
 
 ```bash
-sed -i '' "s/xxx/INSERT-YOUR-AGENT-TOKEN-HERE/g" "$(brew --prefix)"/etc/buildkite-agent/buildkite-agent.cfg
+sed -i '' "s/xxx/INSERT-YOUR-AGENT-TOKEN-HERE/g" "$(brew --prefix)"/user/local/etc/buildkite-agent/buildkite-agent.cfg
```
Comment (triarius) [inline README.md:28]:
```suggestion
sed -i '' "s/xxx/INSERT-YOUR-AGENT-TOKEN-HERE/g" "$(brew --prefix)"/usr/local/etc/buildkite-agent/buildkite-agent.cfg
```

**Review by triarius — APPROVED** [review]

**Comment by triarius** [comment]
Thanks for finding this @bamse16!

**Pull request merged**